sig-scalability/slis/apimachinery_slis.md

```diff
@@ -0,0 +1,199 @@
+# API-machinery SLIs and SLOs
+
+The document was converted from [Google Doc]. Please refer to the original for
+extended commentary and discussion.
+
+***TODO(wojtek-t,deads2k,lavalamp): Get to consensus if we want to define latency
```

Comment:
Last-byte is all that matters for any client receiving lists of things, because the client can't start any work for an object until the last byte arrives.  That'll be true even with paging.